stories/atoms/components/list/List.stories.mdx

```diff
@@ -90,3 +106,162 @@ These expected elements are [ListItem](/docs/atoms-listitem--overview) component
 ## Properties
 
 <ArgsTable story="Overview" />
+
+## Layout
+
+To allow for choice in the list `layout`, you can choose between the following:
+
+- **list** (default): This is a classic list with items being stacked upon each other tightly.
+  It gives the user an easy and fast way of comparing items.
+- **grid**: organizes the items in 350px \* 210px grids. The grid `layout` allow for more detailed list items.
+
+<Canvas>
+  <Story name="Layout">
```

Comment:
Would be good to refactor this code to make it more readable. You could just make a `ListItems` object containing the property values and then do a map for the two different layouts